### ISSUE TITLE
ELISA classified under two branches

### ISSUE THREAD

**Issue by sschurer** [BioAssayOntology/BAO#15]
ELISA is classified under "molecular redistribution" and "binding assessment method"

From Vas:
Remove ELISA from the “molecular redistribution” branch. Most of the time ELISA is performed on lysated cells so the there is really no cell structure there to observe.

Ex situ immunoassay is not a term widely used so I don’t know what exactly does that term entail but will look into it.

IHC could be used under the “molecular redistribution” branch. An easy example to explain immunohistochemistry assays would be: You take and freeze the tissue of interest, slice it in thin layers (with a fancy blade / laser) and take one slice and stain it with antibodies. The slice of the tissue you are observing has preserved the cell structure (its not a cell lysate) so you could see an image of the cells locations and also what antibodies are in which cells etc. 

In summary in would make sense: ELISA => only under "binding assessment method”, IHC => under both “molecular redistribution” and "binding assessment method”.
